doc/Dockerfile

```diff
@@ -16,10 +17,10 @@ RUN apt-get update && apt-get install -y \
     gcc-multilib g++-multilib libc6-dev-i386 lib32ncurses5-dev \
     x11proto-core-dev libx11-dev lib32z1-dev ccache libgl1-mesa-dev \
     libxml2-utils xsltproc unzip bc ninja-build simg2img lz4 zstd \
-    python3-pyelftools python3-crypto libncurses5 libssl-dev -y && \
-    apt-get install --reinstall -y ca-certificates && \
-    pip3 install pycryptodomex && \
-    apt-get clean && rm -rf /tmp/* /var/tmp/*
+    python3-pyelftools python3-crypto libncurses5 libssl-dev
+RUN apt-get install --reinstall -y ca-certificates
+RUN pip3 install pycryptodomex
+RUN apt-get clean && rm -rf /tmp/* /var/tmp/*
```

Comment:
AFAIK, there was a reason to use one very long `RUN` - docker creates a container snapshot after each `RUN`. I don't know why did they make behave it in this way and why it is bad to have multiple snapshots, though. 

https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-which-is-better

Reply:
Thanks for the interesting link.
I have removed the commit with that split of long command into the set of RUN.